adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -66,7 +67,6 @@ private[rdd] object GenomicRDD {
    * Replaces file references in a command.
    *
    * @see pipe
-   *
```

Comment:
I will revert these on a future push. I want to avoid reverting and re-reverting repeatedly.